libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py

```diff
@@ -87,27 +60,6 @@ def test_create_from_objects(
     )
 
 
-def test_create_from_label_objects(
```

Comment:
Remove this test file completely since we are removing bulk import  in V6. This would be complicated to get to work so not worth our time to fix

Reply:
I will have a separate PR on BulkImportRequest